packages/grid/x-grid-modules/src/models/colDef/numericColDef.ts

```diff
@@ -8,7 +8,6 @@ export const NUMERIC_COL_DEF: ColTypeDef = {
   type: 'number',
   align: 'right',
   headerAlign: 'right',
-  width: 80,
```

Comment:
We document 100 as the default, probably also simpler for developers to do the math when they see that they can add or remove space.